maps/away/ships/pra/headmaster/headmaster_ship.dm

```diff
@@ -96,7 +112,13 @@
 /obj/machinery/computer/shuttle_control/explore/headmaster_shuttle
```

Comment:
instead of making this console subtype of `/explore` and setting all the terminal vare, base it on `/explore/terminal`